plugins/primus_lisp/lisp/simple-memory-allocator.lisp

```diff
@@ -29,22 +29,45 @@
 
 (defun memory/allocate (ptr len)
   (if *malloc-initialize-memory*
-      (memory-allocate ptr n *malloc-initial-value*)
-    (memory-allocate ptr n)))
+      (memory-allocate ptr len *malloc-initial-value*)
+    (memory-allocate ptr len)))
+
+(defun word-size () (/ (word-width) 8))
```

Comment:
adding this function to the public interface will introduce confusion, between `word-size` and `word-width` so, please either remove it or mangle correspondingly.